x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -60,8 +55,101 @@ export const renderAsRedirectTo = (to: string) => {
       />
     );
   };
-};
+}
+
+// These component function definitions are used below with the `component`
```

Comment:
An alternative to doing this would be to use nested routes in the `Home`, `Settings`, and `ServiceDetails` components where we can use the location to select the tab. We can explore doing that in a future PR.